code/modules/admin/verbs/randomverbs.dm

```diff
@@ -344,7 +344,7 @@ Traitors and the like can also be revived with the previous role mostly intact.
 		new_character.real_name = record_found.fields["name"]
 		new_character.gender = record_found.fields["sex"]
 		new_character.age = record_found.fields["age"]
-		new_character.hardset_dna(record_found.fields["identity"], record_found.fields["enzymes"], record_found.fields["name"], record_found.fields["blood_type"], new record_found.fields["species"], record_found.fields["features"])
+		new_character.hardset_dna(record_found.fields["identity"], record_found.fields["enzymes"], record_found.fields["name"], record_found.fields["blood_type"], record_found.fields["species"], record_found.fields["features"])

```

Comment:
This won't work, or will probably fix the runtime but will introduce another bug. `hardset_dna()` takes a /datum/species instance for the 5th argument, however the datacore "species" field only contains the species name as per https://github.com/tgstation/tgstation/blob/360fbffac78f71911b2564bc8ecc952a4b6e6d92/code/datums/datacore.dm#L228

Perhaps a speciestype field could be added to the datacore and that could be instantiated here?